packages/core-entities/src/models/views/YieldViews.ts

```diff
@@ -570,9 +599,13 @@ export const YieldViews = (self: Instance<typeof NetworkModel>) => {
 
     const allListedVaultsData = getTokensByType('VaultShare')
     .filter((t) => t.vaultAddress && listedVaultAddresses.includes(t.vaultAddress))
+    .filter((t, index, self) => 
```

Comment:
What is the point of doing this filter?

Reply:
This filter was to remove duplicates of vaultAddresses